app/controllers/scans_controller.rb

```diff
@@ -27,20 +27,11 @@ def new_request_params
   end
 
   def redirect_to_success_with_token
-    redirect_to illiad_query(current_request)
+    redirect_to illiad_url
```

Comment:
I guess I don't know what happens at that URL. Does the user need to do something special (and, is it something we could script) once they get to Illiad?